features/hooks/filtering.feature

```diff
@@ -244,3 +244,186 @@ Feature: filters
       .after context
       """
 
+  Scenario: Filtering hooks using symbols
```

Comment:
Makes sense.  Do you mind putting them in a separate commit?

It helps keeps the history easier to follow when things like this are committed separately -- otherwise it suggests these scenarios depend upon your changes.